academy/xl-cosmjs/intro.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "CosmJs Intro"
+order: 2
+description: What is CosmJs and what can it do for me?
+tag: intro
+---
+
+# The DApp stack
```

Comment:
All h1 headlines should reflect the title of the section. I included a proposed change to make it a better fit format as a suggestion.